testsuite/features/support/constants.rb

```diff
@@ -275,12 +275,14 @@
   %w[
     sle-product-sles15-sp4-pool-x86_64
     sle-manager-tools15-pool-x86_64-sp4
+    sle-manager-tools15-pool-x86_64-sled-sp4
```

Comment:
We currently sync sle-manager-tools15-pool-x86_64-sled-sp3 for some reason in the CI, maybe we should update to  sled-sp4 as well?